doc/design/libsvm_format_column.md

```diff
@@ -0,0 +1,112 @@
+# Support LibSVM format on SQLFlow
+
+## Background
+
+SQLFlow extends SQL syntax to support End-to-End machine learning pipeline. 
+However, due to the variety of the data format (like CSV, LibSVM, etc.), 
+data pre-processing is usually a tough task for users. Users have to speed lots of time 
```

Comment:
It is not the user who transforms the raw inputs.  It is the program that transforms the raw inputs.

Reply:
This sentence has been removed after polishing.